### DISCUSSION TITLE
版本过低或网络不稳定怎么解

### DISCUSSION THREAD

**Discussion by MUYI-XIAN** [simple-robot/Discussions#69] [Q&A] [answered]
### 选择此问题涉及的编程语言

Java

### 疑问描述

一开始能登录，后面写代码时不停重启项目，大概几个小时后就开始说我网络不稳定，查了社区以及相关mira的解决方案
1、更新了组件版本 2、删除了device.json和cache中的账号文件后，使用手机的滑块验证
都无法解决

**Comment by ForteScarlet** [comment, marked as answer]
https://github.com/mamoe/mirai/issues/2496

提示网络不稳定、  `code=235`, `code=237` 等问题属于tx的风控，没有100%有效的解决办法。
在确保版本最新的情况下，如果依旧出现则考虑等待一段时间（例如几小时）或更换其他账号等。
